### PR TITLE
Merge Hijacker into Thief

### PR DESCRIPTION
**Pull request by Smittytron**
As discussed on IRC and [forums](https://forum.openra.net/viewtopic.php?f=82&t=20640), this merges the abilities of the hijacker into the thief.

There is still a hijacker.shp in mods/ra/bits, and I am unsure if that needs to be (or can be) renamed.

### CODE REVIEW

File: mods/ra/rules/disable-player-experience.yaml [hunk 1]
```diff
@@ -22,10 +22,6 @@ THF:
 	Infiltrates:
 		PlayerExperience: 0
 
-HIJACKER:
-	Captures:
```
Comment (abcdefg30) [inline mods/ra/rules/disable-player-experience.yaml]: You will need to disable this on the `THF` now.

**Review by abcdefg30 — COMMENTED** [review]
What is the hijacker shp for? If it is just the icon we can remove it, I guess. (Seems like the hijacker sequence definition used the thf shp anyway?)

**Review by pchote — APPROVED** [review]

**Review by MustaphaTR — APPROVED** [review]
Looks good to me. :+1:

**Comment by MustaphaTR** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/bcd2ab535f638fa467dd22f8d817e008e858d379)

**Pull request merged**